cluster-operator/src/main/java/io/strimzi/operator/cluster/model/StorageDiff.java

```diff
@@ -116,7 +133,88 @@ public boolean changesType() {
      *
      * @return true when the size of the volumes changed
      */
-    public boolean changesSize() {
-        return changesSize;
+    public boolean shrinkSize() {
+        return shrinkSize;
+    }
+
+    /**
+     * Parse a K8S-style representation of a disk size, such as {@code 100Gi},
+     * into the equivalent number of bytes represented as a long.
+     *
+     * @param size The String representation of the volume size.
+     * @return The equivalent number of bytes.
+     */
+    public static long parseMemory(String size) {
```

Comment:
Looks familiar :-)

Reply:
Yeah I had to dig it out from the archives. You happy to see your great code in use again? :-D